homeassistant/components/fritz/binary_sensor.py

```diff
@@ -39,17 +40,17 @@ def __init__(
         """Init FRITZ!Box connectivity class."""
         self._unique_id = f"{fritzbox_tools.unique_id}-connectivity"
         self._name = f"{device_friendly_name} Connectivity"
-        self._is_on = True
+        self._is_on: bool = True
```

Comment:
I think it should infer this automatically.
```suggestion
        self._is_on = True
```